Dockerfile.user

```diff
@@ -7,7 +7,8 @@ ARG GID
 ENV PATH=/vbash/bin:/home/$USER/.local/bin:$PATH
 ENV GOPATH=/home/$USER/gopath
 
-RUN groupadd -g $GID $USER && \
+# Create group if it doesn't exist
+RUN sh -c "if ! getent group $GID; then groupadd -g $GID $USER; fi" && \
```

Comment:
macOS does something differently with groups - the build was failing for me locally with `groupadd: GID '20' already exists`

Reply:
Ah I think the difference here is that the group to which you belong on a Mac host already exists in the Docker base image. Not sure that it's massively relevant however, so totally happy with this change.